library/formatting.inc.php

```diff
@@ -193,6 +199,37 @@ function DateToYYYYMMDD($DateValue)
     }
 }
 
+function TimeToHHMMSS($TimeValue)
+{
```

Comment:
TODO:
Actually, I think this will always just return the input for now (for now easiest to keep input values at HH:MM:SS)
Can improve this down the road, if needed.

Reply:
DONE